src/comment/models.py

```diff
@@ -31,6 +31,5 @@ class Comment(Base):
     )
     author = relationship("Author", back_populates="comment")
 
-    article_id = Column(
-        Integer, ForeignKey("article.id"), name="article_comment"
-    )
+    article_id = Column(Integer, ForeignKey("article.id"), unique=True)
```

Comment:
add `on_delete` here

that `unique=True` is bad. Because article should be able to have many comments, meaning that several comments can have same `article_id`